packages/core/admin/server/services/content-api.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+const _ = require('lodash');
```

Comment:
I think this should live in the content API container next to the permissions property. IMO it's a content-api-related service, not necessarily an admin one

Reply:
Yeah, that makes sense to me too.

Reply:
info would be useful, we want the name, we can get it through different means in the frontend but I don't see why dele would do that, and it's already available (I am not doing anything extra to have this property) I don't see the point of deliberately removing it.

Reply:
If it makes sense because it's needed in the frontend then why not :slightly_smiling_face: